man/less-variables.xml

```diff
@@ -20,10 +20,12 @@
                 <varlistentry id='less'>
                         <term><varname>$SYSTEMD_LESS</varname></term>
 
-                        <listitem><para>Override the default
-                        options passed to
-                        <command>less</command>
-                        (<literal>FRSXMK</literal>).</para></listitem>
+                        <listitem><para>Options to pass to
+                        <command>less</command>. If set override
+                        <varname>$LESS</varname>. If neither
+                        <varname>$LESS</varname> or
+                        <varname>$SYSTEMD_LESS</varname> are set, set it
+                        to <literal>FRSXMK</literal>.</para></listitem>
```

Comment:
This sounds weird. What about this?

Options to pass to `less`. If unset, the environment variables `$SYSTEM_LESS` or, in that order, `$LESS` will be used. If all of these are unset, the options will default to `FRSXMK`.

?